app/src/main/java/com/codesoom/assignment/controllers/AccountController.java

```diff
@@ -0,0 +1,100 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.AccountService;
+import com.codesoom.assignment.domain.Account;
+import com.codesoom.assignment.dto.AccountData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+/**
+ *
+ * 웹에서 회원정보에 대한 요청을 받아 처리 및 출력을 담당합니다.
+ *
+ * @author flying_fisherman
+ */
+
+@RestController
+@CrossOrigin
+@RequestMapping("/products/accounts")
+public class AccountController {
+    private final AccountService accountService;
+
+    public AccountController(AccountService accountService) {
+        this.accountService = accountService;
+    }
+
+    /**
+     * 전체 회원정보를 List화 하여 리턴합니다.
```

Comment:
이렇게 하는 편이 더 낫습니다. `List화 하여`는 굳이 설명할 필요가 없어요. 이런 세부 구현에 대한 정보는 숨깁시다.
```suggestion
     * 전체 회원정보 List를 리턴합니다.
```
```suggestion
     * 전체 회원정보 목록을 리턴합니다.
```